src/main/java/leo/client/Browser.java

```diff
@@ -14,36 +14,33 @@
 import javax.swing.*;
 import java.lang.reflect.Method;
 
-
 public class Browser {
-    private static final String errMsg = "Error attempting to launch web browser";
```

Comment:
Wrong identation?

Reply:
Sorry about that looks like I had a formatter on save.